openmdao/docs/usr-guide/tutorials/external-code-tutorial.rst

```diff
@@ -43,13 +43,16 @@ and writes the output, `f_xy`, to an output file.
 
         input_filename = sys.argv[1]
         output_filename = sys.argv[2]
-        
+
         paraboloid(input_filename, output_filename)
 
 
-Next we need to write the code that makes use of this external code.
 
-::
+Next we need to build the OpenMDAO component that makes use of this external code.
+
+
+.. code-block :: python
```

Comment:
we couldn't use doc tests that actually ran the code, because of the external code nonsense (making systems calls from doctest is a no-no). 

I could let it run on the code instantiation though. Thats a good idea. I guess the expected output would just be nothing.